lib/hsa/hsaPulsarUpdatePhase.cpp

```diff
@@ -8,15 +8,12 @@
 #include "configUpdater.hpp"
 #include "hsaBase.h"
 #include "restServer.hpp"
+#include "visUtil.hpp"
```

Comment:
Ah, was the include "utils/visUtil.hpp" incorrect then? Or would both have worked?

Reply:
This also doesn't have any `[un]likely` uses it in, but I don't really know where the `visUtil.hpp` is coming from.  @nritsche was this included by mistake?

Reply:
I moved the visUtil import from the system includes to the project icludes. I guess both works, but style.

Reply:
Ah I see.